enterprise/cli/provisionerdaemons_test.go

```diff
@@ -35,6 +38,17 @@ func TestProvisionerDaemon_PSK(t *testing.T) {
 	clitest.Start(t, inv)
 	pty.ExpectMatchContext(ctx, "starting provisioner daemon")
 	pty.ExpectMatchContext(ctx, "matt-daemon")
+
+	var daemons []codersdk.ProvisionerDaemon
+	require.Eventually(t, func() bool {
+		daemons, err = client.ProvisionerDaemons(ctx)
+		if err != nil {
+			return false
+		}
+		return len(daemons) > 0
```

Comment:
Can we narrow it to `len(daemons) == 1`?

The question is relevant for other tests too.